sql-aggregates/total-dvds.sql

```diff
@@ -0,0 +1,3 @@
+select count(*)
+from "inventory"
+join "films" using ("filmId");
```

Comment:
You don't need a `join` here, just counting the rows in the `inventory` table will do it.